tests/unit/test_hnsw_multi.cpp

```diff
@@ -1725,21 +1729,5 @@ TYPED_TEST(HNSWMultiTest, markDelete) {
     runBatchIteratorSearchTest(batchIterator, k, verify_res_even);
     VecSimBatchIterator_Free(batchIterator);
 
-    // Unmark the previously marked vectors.
-    for (labelType label = 0; label < n_labels; label++)
-        if (label % 2 == ep_reminder)
-            this->CastToHNSW(index)->unmarkDelete(label);
-
-    ASSERT_EQ(this->CastToHNSW(index)->getNumMarkedDeleted(), 0);
-    ASSERT_EQ(VecSimIndex_IndexSize(index), n + 1);
-
-    // Search for k results around the middle again. expect to find the same results we found in the
-    // first search.
-    runTopKSearchTest(index, query, k, verify_res);
-    runRangeQueryTest(index, query, dim * all_element * all_element, verify_res, k, BY_SCORE);
-    batchIterator = VecSimBatchIterator_New(index, query, nullptr);
-    runBatchIteratorSearchTest(batchIterator, k, verify_res);
-    VecSimBatchIterator_Free(batchIterator);
-
```

Comment:
no need for this?

Reply:
I thought it was redundant after `unmarkDelete` was removed. But on second thought, I'll replace this with reinserting the vectors as I did in HNSW_single.